tensorflow/compiler/xla/service/gpu/kernel_mapping_scheme.h

```diff
@@ -76,19 +76,35 @@ namespace gpu {
 class KernelMappingScheme {
  public:
   enum { DimZ = 0, DimY, DimX, DimTot };
+  // TODO: rename Dilated to Strided?
```

Comment:
The current XLA code is using dilated. I think more people would better understand it if we rename it to Strided.
If you agree, I'll update the code.